vpnglean/glean_parser_ext/templates/macros.jinja2

```diff
@@ -29,7 +29,7 @@ struct {{ obj.name|Camelize }}ExtraParser : EventMetricExtraParser {
       keepStringsAlive.append(s);
       ffiExtras.values[count] = s.constData();
       {% elif type == "boolean" %}
-      ffiExtras.values[count] = parsedExtras._{{item|camelize}} ? "true" : "false";
+      ffiExtras.values[count] = parsedExtras._{{item|camelize}}.toBool() ? "true" : "false";
```

Comment:
I am not sure the new collection is necessary. However, this definitely is :D Thanks for catching it!

Reply:
Initially I used a boolean :)